src/main/java/org/frc5687/rapidreact/subsystems/DriveTrain.java

```diff
@@ -134,7 +137,7 @@ public void updateDashboard() {
 
         metric("BR/Encoder Wheel Vel", _backRight.getWheelVelocity());
         metric("BR/Predicted Wheel Vel", _backRight.getPredictedWheelVelocity());
-
+        DriverStation.reportWarning("Odometry Pose" + getOdometryPose().toString(), false);
```

Comment:
Again, you can shorten this to just `error("Message in here")` thanks to our Outliers subsystem abstract class.